cli/js/lib.deno.ns.d.ts

```diff
@@ -1899,6 +1899,9 @@ declare namespace Deno {
    */
   export const args: string[];
 
+  /** The URL of the file that was originally executed from the command-line. */
+  export const scriptUrl: URL;
+
```

Comment:
Please move this to cli/js/lib.deno.unstable.d.ts

Reply:
Oops, I knew I forgot something…

Reply:
Sorry about the oversight. Done.